test/throw.test.js

```diff
@@ -1,34 +1,34 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const Fastify = require('..')
 
-test('Fastify should throw on wrong options', t => {
+test('Fastify should throw on wrong options', async (t) => {
```

Comment:
there are not async operations
```suggestion
test('Fastify should throw on wrong options', (t) => {
```

Reply:
see commit here:  [c42cb45](https://github.com/fastify/fastify/pull/6002/commits/c42cb450ebf6288cc18501e49ac274a6c199f106)